config/edit/ap.js

```diff
@@ -8,14 +8,15 @@ exports.app_languages = ['en', 'fr', 'es', 'pt']
 exports.app_description = require('./translations.js').translations['app description']
 
 exports.app_storage = 'https://acclabplatforms.blob.core.windows.net/'
+exports.app_suite_url = 'https://acclabs.azurewebsites.net/'
 
 // DESIRED MODULES
 exports.modules = [
 	{ type: 'pads', rights: { read: 0, write: { blank: 4, templated: 1 } } }, // respond IS FOR TEMPLATED PADS
 	{ type: 'pinboards', rights: { read: 0, write: 1 } },
 	{ type: 'templates', rights: { read: 3, write: 3 } },
 	// { type: 'files', rights: { read: 0, write: 1 } },
-	{ type: 'reviews', rights: { read: 2, write: 2, coordinate: 3 }, reviewers: 2 }, // TO DO: UPDATE THIS TO 2
+	{ type: 'my reviews', rights: { read: 2, write: 2, coordinate: 3 }, reviewers: 2 }, // TO DO: UPDATE THIS TO 2
```

Comment:
can't change the internal name -- only the visible text

Reply:
this needs to be only a visual change

Reply:
The view uses the name of the module. Does changing here have any impact on the system?

Reply:
type is used all over the place to identify the modules. we should separate the internal name from the display. can you just add a new field `name` that gets used if it is available? like `(d.name ?? d.type)`. this way we still use type where it's the same but we can change the display name if we want

Reply:
if you search for `d.type === 'reviews'` you find a lot of usages of `'reviews'`

Reply:
better to just create the name field